### PR TITLE
Add icons for folders vs files

### PR DESCRIPTION
**Pull request by bencole12345**
The list of files now has a folder/file icon on the left hand side accordingly.

screenshot_1549558011

I also reworked the UI flow: a normal click takes you directly to viewing a file/folder, while a long-press brings up a dialog with options for deletion and syncing.

### CODE REVIEW

File: app/src/main/java/uk/ac/cam/cl/juliet/fragments/DataFragment.java [hunk 1]
```diff
@@ -59,9 +63,9 @@ public View onCreateView(
         View view = inflater.inflate(R.layout.fragment_data, container, false);
         filesList = view.findViewById(R.id.filesListRecyclerView);
         filesList.setLayoutManager(new LinearLayoutManager(getContext()));
-        DividerItemDecoration divider =
-                new DividerItemDecoration(context, DividerItemDecoration.VERTICAL);
-        filesList.addItemDecoration(divider);
+        //        DividerItemDecoration divider =
+        //                new DividerItemDecoration(context, DividerItemDecoration.VERTICAL);
+        //        filesList.addItemDecoration(divider);
```
Comment (patricoferris) [inline app/src/main/java/uk/ac/cam/cl/juliet/fragments/DataFragment.java]: Should this just be removed entirely?

File: app/src/main/java/uk/ac/cam/cl/juliet/fragments/DataFragment.java [hunk 2]
```diff
@@ -108,36 +112,52 @@ public void onDataFileClicked(
...
+                .setNeutralButton(
+                        "Delete",
+                        new DialogInterface.OnClickListener() {
                             @Override
-                            public void onClick(View v) {
-                                // TODO: sync the file to the server
-                                uploadFile(file, viewHolder);
+                            public void onClick(DialogInterface dialog, int which) {
                                 dialog.cancel();
+                                showConfirmDeleteDialog(file);
```
Comment (patricoferris) [inline app/src/main/java/uk/ac/cam/cl/juliet/fragments/DataFragment.java:144]: Nice haha 👍

File: app/src/main/java/uk/ac/cam/cl/juliet/fragments/DataFragment.java (hunk 1 above)
Comment (bencole12345) [reply app/src/main/java/uk/ac/cam/cl/juliet/fragments/DataFragment.java]: Good spot, I'll do that now!

**Review by patricoferris — APPROVED** [review]
All looking good to me - just checking that the idea will be to add some function to open folders in a new fragment or something?

**Comment by bencole12345** [comment]
That's right, I think it would look nice if we could get an animation in which the contents of the folder slide in from the right side of the screen

**Comment by patricoferris** [comment]
Great work!

**Pull request merged**